static/js/publisher/components/PublishedSnapList/PublishedSnapList.tsx

```diff
@@ -85,12 +76,12 @@ function PublishedSnapList({
     <Strip element="section" shallow>
       <div className="u-fixed-width u-clearfix">
         <h2 className="p-heading--4 u-float-left">My published snaps</h2>
-        <Link
+        <a
```

Comment:
Can this still be `<Link to="…">` as well?
It seems it works for external (non react-router) links (?).